src/X.Serilog.Sinks.Telegram/Batch/Rules/OncePerTimeRule.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading;
+
+namespace X.Serilog.Sinks.Telegram.Batch.Rules;
+
+public class OncePerTimeRule : IRule, IExecutionHook
+{
+    private readonly TimeSpan _delay;
+
+    private DateTime _nextExecution;
```

Comment:
I'm wondering if localdate would behave ok during daylight saving time switch? I never worked with time closely.

Also would this timer well behave if user adjusts system time backward due to time synchronization.

I know that Unix has monotonous clocks for that purpose, that don't correspond to wall clock.